src/components/CytoscapeGraph/graphs/GraphStyles.ts

```diff
@@ -159,6 +160,20 @@ export class GraphStyles {
           opacity: '0.3'
         }
       },
+      {
+        selector: 'node.' + DEGRADED.name,
+        style: {
+          'border-color': DEGRADED.color,
+          'border-width': '2px'
```

Comment:
I think we could consider a wider border to make these nodes more visible, especially when zoomed out in a larger mesh.

Reply:
I'll try with 3px. Anyway that's something easy to fine-tune afterwards.